lib/ex_debug_toolbar.ex

```diff
@@ -112,7 +112,6 @@ defmodule ExDebugToolbar do
   Returns `func` return value.
   """
   @spec record_event(id, String.t(), function()) :: any()
-  @decorate noop_when_toolbar_disabled()
```

Comment:
ahh, right, so start_event and finish_event are noop by themselves. 
makes sense